pkgs/development/python-modules/python-mapnik/default.nix

```diff
@@ -103,7 +104,7 @@ buildPythonPackage rec {
       # import from $out
       rm -r mapnik
     ''
-    + lib.optionalString stdenv.hostPlatform.isDarwin ''
+    + lib.optionalString python.stdenv.hostPlatform.isDarwin ''
```

Comment:
python.stdenv and stdenv should always match

Reply:
This was using `python.stdenv` before as well. I only removed the `let` indirection.
It's not clear to me whether you want this changed and if so to what.